src/ExtensionsSample/ErrorNotifier.cs

```diff
@@ -71,15 +71,17 @@ public void WebNotify(TraceFilter filter)
         /// <param name="filter">The <see cref="TraceFilter"/> that triggered the notification.</param>
         public void EmailNotify(TraceFilter filter)
         {
-            SendGridMessage message = new SendGridMessage()
+            Mail message = new Mail()
             {
                 From = _sendGridConfig.FromAddress,
-                Subject = "WebJob Error Notification",
-                Text = filter.GetDetailedMessage(5)
+                Subject = "WebJob Error Notification"
             };
-            message.To = new MailAddress[] { _sendGridConfig.ToAddress };
+            message.AddContent(new Content("text/plain", filter.GetDetailedMessage(5)));
+            Personalization personalization = new Personalization();
+            personalization.AddTo(_sendGridConfig.ToAddress);
+            message.AddPersonalization(personalization);
 
-            _sendGrid.DeliverAsync(message);
+            _sendGrid.client.mail.send.post(requestBody: message.Get());
```

Comment:
In other places below, I see code like `_sendGrid.SendMessageAsync` being used. What is the difference here?